KeyViewLoops.md

```diff
@@ -0,0 +1,45 @@
+# Key View Loops
+
+### Convention
+It is recommended to let the window automatically calculate your key view loop for you. There are rare cases in which the automatically calculated key view loop is not sufficient.
```

Comment:
can we also add a link to https://cocoadev.github.io/KeyViewLoopGuidelines/ ?

Reply:
Nope! Generally the "rule" has been try to pick a source that's reasonably authoritative and that's unlikely to disappear. I think this rings true for the cocoadev page.